signin.html

```diff
@@ -24,40 +24,47 @@ <h2>
       </h2>
     </div>
     <div class="input-form">
-      <div class="email-input">
-        <form>
-          <ol>
-            <li>
-              <label>
-                이메일 <br>
-                <input type="email">
-              </label>
-            </li>
-            <li class="pwd-input">
-              <label>
-                비밀번호 <br>
-                <input name="focus" type="password">       
-              </label>
-            </li>
-          </ol>
-        </form>
-      </div>
+      <form>
+        <ol>
+          <li class="email-input-wrapper">
+            <label>
+              이메일<br>
+              <input class="email-input error-border" value="" type="email">
+            </label>
+            <p class="error-msg email-blank-error">이메일을 입력해주세요</p>
+            <p class="error-msg email-type-error hide">올바른 이메일 주소가 아닙니다.</p>
+            <p class="error-msg check-email hide">이메일을 확인해주세요</p>
+          </li>
+          <li class="pwd-input-wrapper">
+            <label>
+              비밀번호<br>
+              <input class="pwd-input error-border" value="" type="password">       
+            </label>
+            <button class="pwd-eye-off">
+              <img id="eyeOnOff" src="images/eye-off.svg" alt="eye-off">
+            </button>
+            <p class="error-msg pwd-blank-error">비밀번호를 입력해주세요</p>
+            <p class="error-msg check-pwd hide">비밀번호를 확인해주세요</p>
+          </li>
+        </ol>
+      </form>
     </div>
-    <button class="button-sign"> 로그인 </button>
+    <button class="button-sign" onclick="loginCheck()"> 로그인 </button>
```

Comment:
1. js 파일에서 addEventListener 로 이벤트 등록하신 것도 있던데 여기서는 onClick 속성을 바로 추가하신 이유가 있을까요? 
잘못된 것은 아니지만 어떤 기준을 가지고 구분한 것이 아니라면 하나의 방식으로 통일하는 게 좋을 것 같습니다! 

3. 불필요한 공백 삭제! 
```suggestion
    <button class="button-sign" onclick="loginCheck()">로그인</button>
```